src/source/PeerConnection/PeerConnection.c

```diff
@@ -1127,8 +1140,18 @@ STATUS freePeerConnection(PRtcPeerConnection* ppPeerConnection)
 
     PROFILE_WITH_START_TIME_OBJ(startTime, pKvsPeerConnection->peerConnectionDiagnostics.freePeerConnectionTime, "Free peer connection");
     SAFE_MEMFREE(*ppPeerConnection);
+    *ppPeerConnection = NULL;
```

Comment:
This line isn't needed since it's part of SAFE_MEMFREE.

https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/65e38dac9b30523d43a57bc009d679e627b58d9a/src/common/include/com/amazonaws/kinesis/video/common/CommonDefs.h#L417

Reply:
could we have 
```
if (ppPeerConnection != NULL && *ppPeerConnection != NULL) {
}
```